docs/content/en/reference/template-module.md

```diff
@@ -132,3 +138,66 @@ modules:
 - name: github.com/stencil/example-module
 	version: v1.0.0
 ```
+
+## Releasing a Module
+
+Modules, when generated by the `stencil create` command are configured to release differently based on the current branch.
+
+ * `main` - Creates a pre-release in Github, this is not automatically used or considered in stencil. These releases look like: `v0.0.0-rc.X`
+ * `release` - Creates a release in Github, this is considered automatically when stencil is ran.
+
+### Conventional Commit
+
+By default the versions bumps are made by using the commit messages. We recommend using squash and merge on your repository so that the title will be used for the commit, and thus the PR title is used instead. The rest of this document will assume this is the case.
+
+A standard PR title should follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format. This roughly translates to the format of `type(optionalScope): message`.
+
+
+`feat: support multiple users`
+
+`feat(users): add multiple user support`
+
+Let’s look at each of the different sections of a conventional commit:
+
+#### type
+
+A type controls the what is released, or not released. Generally a type should be specific to the changes in the PR, but when in doubt you can always select one that has the releasing behaviour you want
+
+##### **Major Release (vX.0.0)**
+
+ * **BREAKING CHANGE** - Breaks existing functionality. This should never be used lightly, always talk to #dev-tooling-support before doing this.
+ * **type!** - shorthand for BREAKING CHANGE, use any other type below with a ! at the end.
+
+Examples:
+
+ * `feat!(scope): break all the things`
+ * `feat!: break allllllll the things`
+
+##### **Minor Release (v0.X.0)**
+
+ * `feat` - a feature, this should be something that adds to the service and is end user facing, this should not be a a breaking change.
```

Comment:
either use periods at the end of these bullet points or don't, but just be consistent (look beneath here)